lib/main.ts

```diff
@@ -166,6 +194,9 @@ module compile {
 				settings = fileNameOrSettings || {};
 			}
 
+			finalTransformers = getFinalTransformers(settings.getCustomTransformers);
+			delete settings.getCustomTransformers;
```

Comment:
Using `delete` here will cause a performance problem, we used it before for a similar reason as here. The field can better be removed in `checkAndNormalizeSettings`, see my comment there.